.travis.yml

```diff
@@ -1,10 +1,9 @@
-sudo: false
+
 language: node_js
 node_js:
   - '8'
   - '10'
   - '12'
-  - '14'
```

Comment:
This might be unnecessary to be changed in this PR.

Reply:
Ditto